game/pokemon.go

```diff
@@ -1,10 +1,118 @@
 package game
 
-import "fmt"
+import (
+    "fmt"
+    "math/rand"
+    "time"
+)
 
 type pokemon struct {
+    allCharacters []PokemonCharacter
+    userSelection PokemonCharacter
+}
+
+func (p pokemon) getRandomStrength(min, max int) int {
+    return rand.Intn(max-min+1) + min
+}
+
+func (p pokemon) start(numRounds int) {
+    rand.Seed(time.Now().UnixNano())
+    p.allCharacters = make([]PokemonCharacter, 6)
+    names := []string{"Pikachu", "Blossomite", "Charmander", "Ditto", "Eevee", "Flareon"}
+    // setup characters
+    for i := 0; i < 6; i++ {
```

Comment:
Ask user for how many rounds they want to play instead of always 6

Reply:
They can pick any amount of rounds between 1-6.